package.json

```diff
@@ -21,23 +21,34 @@
     "migrate": "migrate --config conf/migrate.json --template-file migrations/helpers/template.js"
   },
   "contributors": [
-    "Ryan Adams <ryana@learningpool.com>",
+    "Tom Greenfield <Tom.Greenfield@kineo.com>",
```

Comment:
I'd vote to remove the contributors from package.json altogether as per https://github.com/adaptlearning/adapt_framework/pull/2214.